index.bs

```diff
@@ -1118,7 +1118,14 @@ Elected Groups Participation Constraints</h5>
 			the <a href="#random">verifiable random selection procedure</a> described below
 			to choose one person for continued participation and declare the other seat(s) vacant.
 		<li>
-			An individual <em class="rfc2119">must not</em> participate on both the TAG and the AB.
+			The same individual <em class="rfc2119">must not</em> simultaneously serve on any two of
```

Comment:
```suggestion
			The same individual <em class="rfc2119">must not</em> concurrently serve on any two of
```
The two are synonyms, but simultaneity implies instantaneous overlap, whereas concurrent is looser.

Reply:
OK by me.